omplbindings/scenes.go

```diff
@@ -348,41 +342,44 @@ func scene10() *config {
 		r3.Vector{X: 0., Y: 0., Z: -1000.},
 		&spatialmath.R4AA{Theta: 0, RX: 1., RY: 0., RZ: 0.},
 	)
-
-	return &config{
-		Start:      startInput,
-		Goal:       spatialmath.NewPoseFromOrientation(goalPt, startPose.Orientation()),
-		RobotFrame: model,
-		WorldState: &commonpb.WorldState{
-			Obstacles: []*commonpb.GeometriesInFrame{
-				{
-					ReferenceFrame: "world",
-					Geometries: []*commonpb.Geometry{
-						{
-							Center: spatialmath.PoseToProtobuf(pillarPose),
-							GeometryType: &commonpb.Geometry_Box{
-								Box: &commonpb.RectangularPrism{DimsMm: &commonpb.Vector3{
-									X: 130,
-									Y: 130,
-									Z: 2000,
-								}},
-							},
+	worldState, err := referenceframe.WorldStateFromProtobuf(&commonpb.WorldState{
+		Obstacles: []*commonpb.GeometriesInFrame{
+			{
+				ReferenceFrame: "world",
+				Geometries: []*commonpb.Geometry{
+					{
+						Center: spatialmath.PoseToProtobuf(pillarPose),
+						GeometryType: &commonpb.Geometry_Box{
+							Box: &commonpb.RectangularPrism{DimsMm: &commonpb.Vector3{
+								X: 130,
+								Y: 130,
+								Z: 2000,
+							}},
 						},
 					},
 				},
 			},
 		},
+	})
+	if err != nil {
+		return nil, err
 	}
+	return &config{
+		Start:      startInput,
+		Goal:       spatialmath.NewPoseFromOrientation(goalPt, startPose.Orientation()),
+		RobotFrame: model,
+		WorldState: worldState,
+	}, nil
 }
 
 // scene 11: Get the UR5 to hit itself when this set of start and goal data is used for motion planning
 // Corresponds to move that can cause a self-collision on the UR5 from Rand's move set
```

Comment:
Now that this is public, we should probably pull out anything that references specific individuals. My bad for initially putting this in here.

Reply:
Will remove